### ISSUE TITLE
Re-compile Figure S3

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/OpenPBTA-analysis#1498]
S3 panels will need to be re-compiled into an overall figure.

**Comment by sjspielman** [comment]
Note from this comment https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1390#issue-1230185701, the compiled figure needs to be <= 8.5x11" -

> Before submission, we may need to resize some figure compilations to meet the following [guidelines](https://www.cell.com/figureguidelines), namely the 8.5x11" criterion. This is a simple matter of re-exporting from the Graphic app at the given relative size.

**Comment by jharenza** [comment]
Hi @sjspielman - in trying to go through these tables for the manuscript, I realized that we have two "Other CNS" oncoprints - one in Figure 2D and one in S3-A. Currently, both have the same oncoprint, but S3 should have one similar with groups similar to [this](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/b6ae33b3c10651efd3c417d1f3d9172ef894999e/figures/pdfs/supp/figs3/figureS3.pdf). I think some lines are getting crossed with two groupings called "Other". I also noticed (and my bad for not noticing during review), that we have two "Other" rows in the `*n_per_cancer_group.tsv` tables.
